src/renderer/components/MainSideBar/index.js

```diff
@@ -306,6 +315,19 @@ const MainSideBar = () => {
                 isActive={location.pathname === "/swap"}
                 collapsed={secondAnim}
               />
+              {process.env.COMPOUND === "1" ? (
```

Comment:
you should instead `useEnv("COMPOUND")` (but we might drop this as well when we ship in production anyway)